src/cli/domain/package-server-script/bundle/config/index.js

```diff
@@ -0,0 +1,63 @@
+/*jshint camelcase:false */
+'use strict';
+
+var webpack = require('webpack');
+var path = require('path');
+var wrapLoops = require('./wrapLoops');
+var externalDependenciesHandlers = require('./externalDependenciesHandlers');
+
+module.exports = function webpackConfigGenerator(params){
+  return {
+    entry: params.dataPath,
+    target: 'node',
+    output: {
+      path: '/build',
+      filename: params.fileName,
+      libraryTarget: 'commonjs2'
+    },
+    externals: externalDependenciesHandlers(params.dependencies),
+    module: {
+      loaders: [
+        {
+          test: /\.json$/,
+          exclude: /node_modules/,
+          loader: 'json-loader'
+        },
+        {
+          test: /\.js?$/,
+          exclude: /node_modules/,
+          loaders: [
+            'falafel-loader',
+            'babel-loader?' + JSON.stringify({
+              cacheDirectory: true,
+              'presets': [
+                [require.resolve('babel-preset-env'), {
+                  'targets': {
+                    'node': 0.10
```

Comment:
I know why we are doing this but is there a plan to move away from 0.10... LTS has even ran out on it: https://github.com/nodejs/LTS